content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -684,13 +746,21 @@ Here are a few things to check for:
 - **Is the auto-instrumentation for the right language?** For example, when
   instrumenting a Python application, make sure that the annotation doesn't
   incorrectly say `instrumentation.opentelemetry.io/inject-java: "true"`
-  instead.
+    instead.
```

Comment:
Why this whitespace change? It should be undone, though the formatter should have fixed / reported it.